docs/ROADMAP.md

```diff
@@ -2,30 +2,29 @@
 
 This document contains tentative plans for changes and improvements of note in upcoming versions of the HELICS library. All dates are approximate and subject to change, but this is a snapshot of the current planning thoughts. See the [projects](https://github.com/GMLC-TDC/HELICS/projects) for additional details
 
-## \[2.8.1\] ~ 2022-03-01
+## \[2.8.1\] ~ 2022-05-20
 
 This will be the last of the 2.X series releases
 
 Mainly includes any relevant bug fixes and other minor tweaks for compatibility, and a final refresh of the third-party libraries to bring them up to date.
 
-## \[3.2\] ~ 2022-02-15
+## \[3.2.1\] ~ 2022-05-28
 
-Mostly things that didn't make it into the 3.1 release and a number of bug fixes that come from transitioning to HELICS 3.0. Also including translators which is a late breaking feature coming from many discussions on domain API's and other capabilities.
+Mostly refinement of the 3.2 release and additional testing a bug fixes
```

Comment:
```suggestion
Mostly refinement of the 3.2 release and additional testing and bug fixes
```

Reply:
can merge this fix directly, there is a another PR incoming with this change